### DISCUSSION TITLE
Add short flags for npx

### DISCUSSION THREAD

**Discussion by basickarl** [npm/feedback#624] [CLI]
https://www.npmjs.com/package/npx#description

Currently if I wish to use `--no-install` I need to write  `--no-install`. It would be nice to have short flags for these.

**Comment by ljharb** [comment]
In npm 7+, `npx` isn't that package, it's just `npm exec`, which "does not install" by default.

**Comment by basickarl** [comment]
Ah thanks for the info, missed that. For others read more here: https://docs.npmjs.com/cli/v8/commands/npm-exec
